README.md

```diff
@@ -13,6 +13,22 @@ Helicopter Server is a project for a causal tree access management system. The s
 - Protoc compiler
 - Go and gRPC plugins for Protoc
 
+## Try with examples!
```

Comment:
"Try it out with examples" or maybe "Experiment with these examples" sounds better imo